docker-compose.yml

```diff
@@ -351,7 +351,7 @@ services:
   stable24:
     image: ghcr.io/juliushaertl/nextcloud-dev-php${PHP_VERSION:-74}:latest
     environment:
-      SQL: ${SQL:-mysql}
+      SQL: 'sqlite'
```

Comment:
why are you defaulting to sqlite here?

Reply:
See #90 for the discussion but let's keep such changes out of the general documentation refactoring and rather move that into follow ups then if there are adjustments needed.

We should still then figure out how to make mysql work out of the box (which does at least for me)

Reply:
> why are you defaulting to sqlite here?

In fact, I cannot use `mysql` for the `stable24` service.

So, I/we have to fin a solution to fix this problem... :thinking: 


> See https://github.com/juliushaertl/nextcloud-docker-dev/pull/90 for the discussion but let's keep such changes out of the general documentation refactoring and rather move that into follow ups then if there are adjustments needed.
>
> We should still then figure out how to make mysql work out of the box (which does at least for me)

Yeah, I refactor this part... I don't know why I change this line in the docker-compose file :thinking: 

I don't understand @juliushaertl . Can you use `mysql` with the `stable24` service without problems?

Reply:
For me mysql with stable24 worked without any problems so far

Reply:
> I don't understand @juliushaertl . Can you use mysql with the stable24 service without problems?

Yes, but let's continue in #90 with the topic then

Reply:
I reverted the mysql database by default for stable24 in this branch.
We continue to see that in #90  :wink: